### PR TITLE
MTL-1607 Adopt new kernel to fix CVE-2022-0185 [1.0]

### PR DESCRIPTION
**Pull request by mtupitsyn**
# Summary and Scope

Bump kernel to 5.3.18-24.99.1 to adopt fix for CVE-2022-0185.

https://www.suse.com/security/cve/CVE-2022-0185.html

Fixes [MTL-1607](https://jira-pro.its.hpecorp.net:8443/browse/MTL-1607)

## Issue Type

Bugfix Pull Request

### CODE REVIEW

**Review by heemstra — APPROVED** [review]

File: boxes/sles15-base/scripts/kernel.sh [hunk 1]
```diff
@@ -1,8 +1,29 @@
...
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+SLES15_KERNEL_VERSION="5.3.18-24.99.1"
 
-KERNEL_PACKAGES=(kernel-default-debuginfo-"$SLES15_KERNEL_VERSION"
-kernel-default-devel-"$SLES15_KERNEL_VERSION"
-kernel-default-debugsource-"$SLES15_KERNEL_VERSION"
+KERNEL_PACKAGES=(kernel-default-devel-"$SLES15_KERNEL_VERSION"
```
Comment (mtupitsyn) [inline boxes/sles15-base/scripts/kernel.sh:26]: For patched kernel, coming with LTSS subscription, kernel-default-debuginfo and kernel-default-debugsource packages are not available. We'll have to either install closest available version (5.3.18-24.83.2) or not install them at all.

**Comment by mtupitsyn** [comment]
Will convert it to draft until build issues are resolved.

**Review by heemstra — APPROVED** [review]

**Pull request merged**